packages/foreman/nodejs-theforeman-builder/nodejs-theforeman-builder.spec

```diff
@@ -177,114 +181,118 @@ BuildArch: noarch
-Provides: bundled(npm(@babel/plugin-transform-reserved-words)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-shorthand-properties)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-spread)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-sticky-regex)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-template-literals)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-typeof-symbol)) = 7.8.4
-Provides: bundled(npm(@babel/plugin-transform-unicode-regex)) = 7.8.3
-Provides: bundled(npm(@babel/preset-env)) = 7.9.0
+Provides: bundled(npm(@babel/plugin-transform-new-target)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-object-assign)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-object-super)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-parameters)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-property-literals)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-react-display-name)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-react-jsx)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-react-jsx-development)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-react-jsx-self)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-react-jsx-source)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-react-pure-annotations)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-regenerator)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-reserved-words)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-shorthand-properties)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-spread)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-sticky-regex)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-template-literals)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-typeof-symbol)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-unicode-escapes)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-transform-unicode-regex)) = 7.10.1
+Provides: bundled(npm(@babel/preset-env)) = 7.10.2
 Provides: bundled(npm(@babel/preset-modules)) = 0.1.3
-Provides: bundled(npm(@babel/preset-react)) = 7.9.4
-Provides: bundled(npm(@babel/runtime)) = 7.9.2
-Provides: bundled(npm(@babel/template)) = 7.8.6
-Provides: bundled(npm(@babel/traverse)) = 7.9.0
-Provides: bundled(npm(@babel/types)) = 7.9.0
-Provides: bundled(npm(@theforeman/builder)) = 4.3.0
+Provides: bundled(npm(@babel/preset-react)) = 7.10.1
+Provides: bundled(npm(@babel/runtime)) = 7.10.2
+Provides: bundled(npm(@babel/template)) = 7.10.1
+Provides: bundled(npm(@babel/traverse)) = 7.10.1
+Provides: bundled(npm(@babel/types)) = 7.10.2
+Provides: bundled(npm(@theforeman/builder)) = 4.7.0
 Provides: bundled(npm(ansi-styles)) = 3.2.1
-Provides: bundled(npm(babel-plugin-dynamic-import-node)) = 2.3.0
-Provides: bundled(npm(browserslist)) = 4.11.1
-Provides: bundled(npm(caniuse-lite)) = 1.0.30001038
+Provides: bundled(npm(babel-plugin-dynamic-import-node)) = 2.3.3
```

Comment:
This version caused core build to fail - https://projects.theforeman.org/issues/29605, and is pinned in core package.json. Will this upgrade break our ability to build core?

Reply:
I'm not why it has those 2 lines after running the script:
```
Provides: bundled(npm(babel-plugin-dynamic-import-node)) = 2.3.0
Provides: bundled(npm(babel-plugin-dynamic-import-node)) = 2.3.3
```

When running `npm ls babel-plugin-dynamic-import-node` in foreman, I am getting the expected results:
```
TheForemanDevDeps@2.2.0 /home/vagrant/foreman
├─┬ @theforeman/builder@4.7.1
│ ├─┬ @babel/plugin-transform-modules-amd@7.9.0
│ │ └── babel-plugin-dynamic-import-node@2.3.0  deduped
│ ├─┬ @babel/plugin-transform-modules-commonjs@7.9.0
│ │ └── babel-plugin-dynamic-import-node@2.3.0  deduped
│ ├─┬ @babel/plugin-transform-modules-systemjs@7.9.0
│ │ └── babel-plugin-dynamic-import-node@2.3.0  deduped
│ └── babel-plugin-dynamic-import-node@2.3.0
└─┬ @theforeman/test@4.7.1
  └── babel-plugin-dynamic-import-node@2.3.0  deduped
```

Any suggestion?

Reply:
@evgeni you might have a clue